packages/slice-machine/src/modules/useSliceMachineActions.ts

```diff
@@ -48,7 +48,9 @@ const useSliceMachineActions = () => {
       getEnvironmentCreator({
         env: serverState.env,
         warnings: serverState.warnings,
-        configErrors: serverState.configErrors,
+        configErrors: !!serverState.configErrors
```

Comment:
I think you adapted this part to always have an object for this but you also changed the type of serverState yourself.
And I didn't saw any changes when affecting the state itself.

Reply:
Fixed :)